stdlib/2and3/netrc.pyi

```diff
@@ -0,0 +1,19 @@
+from typing import AnyStr, Dict, List, Optional, Tuple, overload
+
+
+class NetrcParseError(Exception):
+    filename: Optional[str]
+    lineno: Optional[int]
+    msg: str
+
+
+# (login, account, password) tuple
+_NetrcTuple = Tuple[str, Optional[str], Optional[str]]
+
+
+class netrc:
+    hosts: Dict[str, _NetrcTuple]
+    macros: Dict[str, List[str]]
+
+    def __init__(self, file: str=...) -> None: ...
```

Comment:
PEP 8 recommends spaces around default arg `=` if there is an annotation.

Reply:
Fixed.
My experience with typing is very tiny, thanks for your hints.